### ISSUE TITLE
AndroidManifest merge conflict because of android:label="@string/app_name"

### ISSUE THREAD

**Issue by mosofsky** [tomeeeS/ScrollPicker#2]
[ScrollPicker's AndroidManifest.xml](https://github.com/tomeeeS/ScrollPicker/blob/master/src/main/AndroidManifest.xml) contains:

```
    <application
        android:label="@string/app_name"
        android:supportsRtl="true"
    >
```

But my AndroidManifest.xml also refers to `android:label`:

```
    <application
        android:icon="@mipmap/locuslabs_app_icon"
        android:label="AirportList"

```

So when I try to compile my app I get the following error:

```
ERROR: Manifest merger failed : Attribute application@label value=(AirportList) from AndroidManifest.xml:23:9-36
	is also present at [com.github.tomeees:scrollpicker:1.7.3] AndroidManifest.xml:12:9-41 value=(@string/app_name).
	Suggestion: add 'tools:replace="android:label"' to <application> element at AndroidManifest.xml:21:5-46:19 to override.
```

Indeed, adding `tools:replace="android:label"` to the `<application>` tag of my AndoidManifest.xml works around the problem.

However, I've incorporated ScrollPicker into a library so all of the users of my library will have to implement the same workaround.

If `android:label` isn't needed by ScrollPicker, I suggest removing it.  As justification, consider that an alternative [Stepper-Touch](https://github.com/DanielMartinus/Stepper-Touch/blob/master/library/src/main/AndroidManifest.xml) doesn't use `android:label` in its AndroidManifest.xml.

ScrollPicker is a great piece of code, my suggestion would just be helpful to its users.  Thank you for making ScrollPicker.

**Comment by tomeeeS** [comment]
Hi, thanks for telling me and glad you like it!
Have just fixed it and released 1.7.4.

**Comment by mosofsky** [comment]
Thank you @tomeeeS , we upgraded to 1.7.4 and confirmed it resolve the issue.
